setup/productionize/scaling/horizontal-scaling.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: single
+title:  "Horizontally Scale Spinnaker Services"
+sidebar:
+  nav: setup
+---
+
+Spinnaker's services are intended to be replicated for performance and
+availability. This page will explain under which circumstances it can help
+to run more than one copy of any single service, and what considerations
+have to be made when doing so.
+
+It is assumed that the reader can scale/provision more replicas of a service
+either by tweaking the [distributed Kubernetes installation's
+component-sizing](/reference/halyard/component-sizing/#replicas), or via
+whatever automation they chose to deploy Spinnaker services on VMs. The
```

Comment:
I'd remove `on VMs` here, because someone might be running Spinnaker in a non-VM environment without Halyard.

Reply:
Doesn't that mean we should call that out too?

Reply:
How 'bout neither?

"This article assumes you have access to scale the infrastructure running Spinnaker, and the mechanics of doing that are not covered here.

Reply:
sg